Ntuplizer/test/ntuplize_cfg.py

```diff
@@ -224,6 +237,9 @@
 from UWVV.AnalysisTools.templates.ElectronScaleFactors import ElectronScaleFactors
 FlowSteps.append(ElectronScaleFactors)
 
+from UWVV.AnalysisTools.templates.AddMetFilters import AddMetFilters 
```

Comment:
This should probably only be done for WZ, we don't need it in ZZ.

Reply:
I would argue that including these is a good idea for most analyses. They aren't actually filters on the event, just flags which are recommended for the analysis. It is recommended for any case where MET is used, i.e. the ZZ fake rate. Of course, if another analysis wants it off they can remove it. The badMuonFilter is also included.

Reply:
So no events are actually filtered, it's just some items in the ntuples one could use to filter? That's ok I guess, as long as it's not going to increase the runtime by a lot for everyone else.

Reply:
Right, its run in a mode to just add boolean values.